### DISCUSSION TITLE
Home Assistant OS 14.0.rc3

### DISCUSSION THREAD

**Discussion by sairon** [home-assistant/operating-system#3709] [Announcements]
## Home Assistant Yellow

* Disable CM5's RTC to make Yellow's on-board RTC the primary one (#3706) @sairon
* Disable CQE on mmc0 to fix I/O freezes on Yellow+CM5 (#3705) @sairon
* Reduce time of mmc0 interface random hangs on Yellow+CM5 boot (#3700) @sairon

## Home Assistant Green

* Linux: Update kernel to 6.6.63 (#3697) @sairon

## Open Virtual Appliance

* Linux: Update kernel to 6.6.63 (#3697) @sairon

## Generic x86-64

* Linux: Update kernel to 6.6.63 (#3697) @sairon

## Hardkernel ODROID

* Linux: Update kernel to 6.6.63 (#3697) @sairon

## ASUS Tinker

* Linux: Update kernel to 6.6.63 (#3697) @sairon

## Khadas VIM Series

* Linux: Update kernel to 6.6.63 (#3697) @sairon

## Generic aarch64

* Linux: Update kernel to 6.6.63 (#3697) @sairon

## Build System

* Bump docker/build-push-action from 6.9.0 to 6.10.0 (#3704) @dependabot

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/14.0.rc3'>Home Assistant OS 14.0.rc3</a>.</em>

**Comment by Anto79-ops** [comment]
There are no issues on my NUC 12 (x86-64, bare-metal) with USB SkyConnect and USB Aeotec Z-Stick 7.

The system is running normally.
